tests/integration/update_cluster/ha_gce/data/google_compute_instance_template_master-us-test1-a-ha-gce-example-com_metadata_startup-script

```diff
@@ -123,8 +123,6 @@ ensure-install-dir
 
 cat > conf/cluster_spec.yaml << '__EOF_CLUSTER_SPEC'
 cloudConfig:
-  gcpPDCSIDriver:
```

Comment:
We might need to embed the GCE config so we know when to restart instances?

Reply:
The only thing in there is an "enabled" flag. If there were config, it would go into the addon manifest, which I don't think needs an instance restart to be applied. If we add something that requires nodeup to change its build-out behavior, I'd hope we add that setting to nodeup.Config.

Reply:
The GCE service account goes into the ASGs and the bucket ACLs. I would think the former would cause instances to be marked for rolling update.

Reply:
OK - sgtm!  Merge and iterate :-)